script.skinshortcuts/addon.xml

```diff
@@ -37,4 +38,8 @@
 		<email></email>
 		<source>https://github.com/BigNoid/script.skinshortcuts</source>
 	</extension>
+	<assets>
+		<icon>icon.png</icon>
+		<fanart>fanart.jpg</fanart>
+	</assets>
```

Comment:
this probably slipped in :-)
assets tag should be part "xbmc.addon.metadata" extension point,
and will only work for kodi krypton and up.

```15:23:35 T:6144569344   DEBUG: ADDON: cpluff: 'Suspicious plug-in descriptor content in /var/mobile/Containers/Data/Application/69598EFF-C2DD-4D78-8128-64E076F4A72D/Library/Caches/home/addons/script.skinshortcuts/addon.xml, line 41, column 2 (ignoring unexpected element assets and its contents).'```